### ISSUE TITLE
[api-extractor] Improve how relative paths are resolved in api-extractor.json 

### ISSUE THREAD

**Issue by octogonz** [Microsoft/web-build-tools#1017]
PR https://github.com/Microsoft/web-build-tools/pull/1016 introduced a script that generates multiple different **api-extractor.json** config files in order to process different entry points.  Because these files aren't in the project root, it revealed inconsistencies in how paths are resolved:

- relative to the project folder (containing **package.json**)
- relative to the folder containing **api-extractor.json** (or a secondary config file referenced via `"extends"`?)
- relative to the shell's current working directory when the tool is invoked

We should make the path resolution consistent, and design it to allow the same basic **api-extractor.json** to be reused against different entry points.

In short, we should be able to entirely eliminate [this code](https://github.com/Microsoft/web-build-tools/blob/63ce710bcdee7a3b4904048060216bfbe011abb6/build-tests/api-extractor-scenarios/src/runScenarios.ts#L28)

**Comment by octogonz** [comment]
Fixed by PR https://github.com/Microsoft/web-build-tools/pull/1206
